apps/webapp/components/layout/header/index.tsx

```diff
@@ -14,7 +14,7 @@ import { Navigation } from './new-nav'
 
 export function Header({ lang, dict }: HeaderProps) {
   return (
-    <div className="sticky top-0 z-50 bg-background md:mt-8">
+    <header className="sticky top-0 z-50 bg-background md:pt-4 -top-4">
```

Comment:
I forgot that detail for the new top className: add the `md` breakpoint so on mobile won't look cut 😬

Reply:
Done, top-0 works on the other ones, and md:top-4 will handle the rest. 

Note: I found another issue (see images). Tittle was too big, so I did a little change to fix it 

Big Tittle:
![Captura de pantalla 2024-08-26 a la(s) 1 38 18 p  m](https://github.com/user-attachments/assets/92db0ea6-aa4a-4607-a677-e354e894c02a)
![Captura de pantalla 2024-08-26 a la(s) 1 43 29 p  m](https://github.com/user-attachments/assets/29d56bf5-5d25-42e2-8347-ce51b5b76b7b)

Fixed V:
![Captura de pantalla 2024-08-26 a la(s) 1 46 48 p  m](https://github.com/user-attachments/assets/035c3333-4845-4c2e-b3c7-9775a1ec4d57)